index.html

```diff
@@ -6,14 +6,1394 @@
+                    height="417" srcset="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+                    style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/Animal-Crossing-New-Horizons-796x417.jpg&quot;);">
+                </div>
+                <a class="cover-trigger"
+                    href="https://thenextweb.com/gaming/2020/05/08/detroit-lions-use-animal-crossing-reveal-2020-schedule/"
+                    data-event-category="Homepage" data-event-action="Top story"
+                    data-event-label="Detroit Lions use Animal Crossing (yes, really) to reveal 2020 schedule"
+                    data-event-non-interaction="true"></a>
+                <div class="cover-gradient">
+
+                </div>
+                <div class="cover-logo"></div>
+                <div class="cover-date">
+                    Wednesday — May 6, 2020
+                </div>
+
+                <header class="cover-header">
+                    <h5 class="cover-section"><a href="#">Gear</a></h5>
+                    <h2 class="cover-title">
+                        <a href="#">
+                            Comparing Huawei’s FreeBuds 3i to AirPods is fair, but misguided
+                        </a>
+                    </h2>
+                    <ul class="cover-byline">
+                        <li>by <a href="#">Callum Booth</a></li>
+                        <li><i class="far fa-clock cover-icon"></i>
+                            <time datetime="2020-05-06T13:18:23+00:00" class="time-ago">32 minutes ago</time></li>
+                        <li><i class="fas fa-share-alt cover-icon"></i>0</li>
+                    </ul>
+                </header>
+            </article>
+            <article class="cover-story second-cover">
+                <div class="cover-image lazy lazyLoaded" sizes="(max-width: 796px) 100vw, 796px" width="796"
+                    height="417" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
+                    style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/04/rbcc-web_ad-feature_img-1.jpg&quot;);background-size: cover;">
+                </div>
+                <a class="cover-trigger" href="https://thenextweb.com/events/rebrand" target="_blank"
+                    rel="noopener noreferrer" data-event-category="Homepage" data-event-action="Top story"
+                    data-event-label="Headspace, Airbnb &amp; Google share brand and digital marketing strategy at Re:Brand online event"
+                    data-event-non-interaction="true"></a>
+                <header class="cover-header">
+                    <h2 class="cover-title">
+                        <a href="#">
+                            Headspace, Airbnb & Google share brand and digital marketing strategy at Re:Brand online
+                            event
+                        </a>
```

Comment:
The letters on the side articles are cropped when reducing the size. Kindly fix that.

> Screenshot from your page.
![image](https://user-images.githubusercontent.com/57414789/81483019-b4407580-9200-11ea-92fa-6b1be98cb05c.png)